docs/core/tools/dotnet-vstest.md

```diff
@@ -5,315 +5,151 @@ ms.date: 05/30/2018
 ---
 # dotnet vstest
 
-[!INCLUDE [topic-appliesto-net-core-all](../../../includes/topic-appliesto-net-core-all.md)]
+**This article applies to:** ✔️ .NET Core 2.1 SDK and later versions
 
 ## Name
 
 `dotnet-vstest` - Runs tests from the specified files.
 
 ## Synopsis
 
-<!-- markdownlint-disable MD025 -->
-
-# [.NET Core 2.1](#tab/netcore21)
-
 ```dotnetcli
-dotnet vstest [<TEST_FILE_NAMES>] [--Settings|/Settings] [--Tests|/Tests] [--TestAdapterPath|/TestAdapterPath]
-    [--Platform|/Platform] [--Framework|/Framework] [--Parallel|/Parallel] [--TestCaseFilter|/TestCaseFilter] [--logger|/logger]
-    [-lt|--ListTests|/lt|/ListTests] [--ParentProcessId|/ParentProcessId] [--Port|/Port] [--Diag|/Diag] [--Blame|/Blame] [--InIsolation|/InIsolation]
+dotnet vstest [<TEST_FILE_NAMES>] [--Settings|/Settings]
+    [--Tests|/Tests] [--TestAdapterPath|/TestAdapterPath]
+    [--Platform|/Platform] [--Framework|/Framework]
+    [--Parallel|/Parallel] [--TestCaseFilter|/TestCaseFilter]
+    [--logger|/logger] [-lt|--ListTests|/lt|/ListTests]
+    [--ParentProcessId|/ParentProcessId] [--Port|/Port]
+    [--Diag|/Diag] [--Blame|/Blame] [--InIsolation|/InIsolation]
```

Comment:
I think for this and the sections below, we should just kill the `/parameter` flavors. The `-` and `--` are more cross-platform friendly and having the `/` versions just makes things noisy without any real benefit.

Also, a  lot of the parameter descriptions below are using `-` when they should be using `--`.